menu.js

```diff
@@ -1,17 +1,215 @@
+      technologies: ["Ruby on Rails", "CSS", "JavaScript", "HTML"],
+      live: "https://github.com/0sugo/portfolio_mobile_view",
+      source: "hhttps://0sugo.github.io/portfolio_mobile_view/",
+      index:"six",
+    },
+  ];
+
+  const recentWorks = document.querySelector(".recent-works");
+
+
+  const topper = document.createElement("div");
+  topper.classList.add("topper");
+  recentWorks.appendChild(topper);
+
+  const section_head = document.createElement("h2");
+  section_head.innerHTML = "My Recent Works";
+  section_head.setAttribute('id', 'topper_head');
+  recentWorks.appendChild(section_head);
+
+  const indicator = document.createElement("div");
+  indicator.classList.add("indicator");
+  recentWorks.appendChild(indicator);
+
+  const recentWorkCards = document.createElement("div");
+  recentWorkCards.classList.add("recent-works-cards");
+  recentWorks.appendChild(recentWorkCards);
+
+  cards.forEach((card) => {
+    let works = document.createElement("div");
+    works.classList.add("works", "work1");
+    recentWorkCards.append(works);
+
+    const workDetails = document.createElement("div");
+    workDetails.classList.add("work-details");
+    works.append(workDetails);
+
+    const title = document.createElement("h3");
+    title.innerHTML = `${card.name}`;
+    workDetails.append(title);
+
+    const languageTags = document.createElement("ul");
+    languageTags.classList.add("language-tags");
+    workDetails.append(languageTags);
+
+    for (let i = 0; i < card.tech.length; i++) {
+      const skill = document.createElement("li");
+      skill.innerHTML = `${card.tech[i]}`;
+      languageTags.append(skill);
+    }
+
+    const see = document.createElement("a");
+    see.innerHTML = "see project";
+    see.setAttribute('href', `#${card.id}`);
+    see.addEventListener("click",modal);
+    see.classList.add("works-button");
+    workDetails.appendChild(see);
+    
+
+  });
+  const modal_pop = document.querySelector(".try");
+
+  function modal() {
+    let pops = '';
+    projects.forEach((project) => {
+      pops += `<div id="${project.index}" class="overlay">
+<div class="popup">
+                <a class="close" href="#">&times;</a>
+                <div class="content">
+                    <div class="pop-image"></div>
+
+                    <div class="contented">
+                    <h2 id="item-1">${project.name}</h2>
+                    <ul class="language-tag" id="item-2">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <p id="item-3">${project.description}</p>
+
+                    <div class="see-section" id="item-4">
```

Comment:
### OPTIONAL

- [ ] Kindly use ` justify-content: space-between;`, this will  align the content better and make  your design look like the Figma design 👍🏿